src/actions/index.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  addBook: (state, action) => (
+    {
+      ...state,
+      books: [...state.books, action.payload],
+    }
```

Comment:
This kind of work belongs to the reducer an action should look like:
```
const addBook = book => ({
  type: "CREATE_BOOK",
  book,
});

```

Reply:
Made the necessary adjustments to make the actions look like what it was told to us.